astro/src/content/docs/_shared/_idp-form-fields.astro

```diff
@@ -10,10 +10,13 @@ const { getStartedHref, getStartedText, idp, linkingStrategy, scopeDefault, scop
 <h3>Form Fields</h3>
 
 <APIBlock>
-  <APIField name="Id" optional since="1.61.0">
+  <APIField name="Id" optional since="1.61.0" renderif={idp !== 'Azure'}>
     An optional UUID. When this value is omitted a unique Id will be generated automatically.
```

Comment:
```suggestion
    An optional UUID. When this value is omitted a unique ID will be generated automatically.
```
[suggestion]: not your mistake, but might as well fix this in this PR since I  have related suggestions below.